string group_name
JointMsg[] joints
geometry_msgs/Pose[] poses_left
geometry_msgs/Pose[] poses_right
uint32 total_joints
uint32 total_points